### ISSUE TITLE
[v4 β1] ChipField text color regression when used as a child of ReferenceField

### ISSUE THREAD

**Issue by smeng9** [marmelab/react-admin#7278]
**What you were expecting:**
The text color should be default anchor link color

**What happened instead:**
The text color is black, so it does not respect anchor link color even though it is a link.

**Steps to reproduce:**
See codesandbox below.

Just replace some TextField in ReferenceField with ChipField.

**Related code:**

* Preferably, a CodeSandbox forked from https://codesandbox.io/s/github/marmelab/react-admin/tree/master/examples/simple
* A link to a GitHub repo with the minimal codebase to reproduce the issue

```
[3.19.10](https://codesandbox.io/s/jolly-hopper-egf7x8?file=/src/invoices/InvoiceList.tsx:1538-1581)
[4.0.0-beta.1.0](https://codesandbox.io/s/nostalgic-frog-ct4r1f?file=/src/posts.js:689-725)
```

**Other information:**

![chipfield](https://user-images.githubusercontent.com/38666763/154771913-9157b29b-2efd-4c6d-a33f-c43c26626549.PNG)
![color2](https://user-images.githubusercontent.com/38666763/154771924-d329fbbc-f925-4ab4-a9fa-1d5bc91b8833.PNG)

Suggest: to make color inherit from parent. Still not sure why in v3 even without inheriting the link text color it works out of the box.

**Environment**

* React-admin version: 4.0.0-beta.1.0
* Last version that did not exhibit the issue (if applicable): 3.19.10
* React version: 18.0.0-rc.0
* Browser: Edge 98
* Stack trace (in case of a JS error): N/A for css regression

**Comment by WiXSL** [comment]
Reproduce. Thanks

**Comment by fzaninotto** [comment]
Fixed by #7283
